exercises/practice/dnd-character/dnd-character_spec.lua

```diff
@@ -122,7 +122,6 @@ describe('dnd', function()
       assert.equal("integer", math.type(strength))
       assert.lteq(3, strength)
       assert.lteq(strength, 18)
-      assert.equal(strength, character.strength)
```

Comment:
This check is unnecessary because `strength` is initialized with `character.strength` above.

Reply:
The [problem-specifications](https://github.com/exercism/problem-specifications/blob/8175ac1ffcbad1424e6ceac865bf8a389b04b489/exercises/dnd-character/canonical-data.json#L196) say to check "each ability is only calculated once"

Here is a solution we want to reject:

```
function Character:new(name)
  self.__index = function(mytable, key)
    return ability(roll_dice())
  end

  local meta_tbl = {
    name = name,
    constitution = ability(roll_dice())
  }
  meta_tbl.hitpoints = 10 + modifier(meta_tbl.constitution)

  return setmetatable(meta_tbl, self)
end
```


Instead of 3 tests
```
      "description": "random ability is within range",
      "description": "random character is valid",
      "description": "each ability is only calculated once",
```
I wrote one test for each ability.

Reply:
Thanks, I put them back.